cli/tests/testdata/compile/workers/basic.ts

```diff
@@ -0,0 +1,11 @@
+import "./worker.ts";
```

Comment:
This is strange - effectively both main worker and web worker will attach event listeners if users don't use `import.meta.main` check. That seems a bit error prone. 

That said, being able to add web workers to `deno compile`d applications is one of the most requested features, so maybe we should do just that.

Reply:
See https://github.com/denoland/deno/pull/17657#issuecomment-1426812661. And even if #17663 doesn't end up landing, denoland/eszip#115 already allows [this hack](https://github.com/denoland/deno/pull/17663#issuecomment-1426832462) to have workers included in the statically analyzed module graph.